docs/Release_notes.md

```diff
@@ -1,6 +1,69 @@
 CARMA System Release Notes
 ----------------------------
 
+Version 4.7.0, released Aug 26th, 2024
+----------------------------------------
+
+### **Summary**
+This release introduces significant enhancements and new functionalities of CDA-Telematics tool, focusing on improving data streaming, integration and visualization capabilities. Key updates include support for streaming data from the XIL environment, integration of telematics tools with V2X Hub and enhanced UI visualization with historical ROS 2 data. Additionally, telematics tool for local deployment process is improved to facilitate use with tools like CAVe-in-a-box. Along with these enhancements, several bug fixes and deployment related enhancements are included in this release.
+
+### **CDA-Telematics** 
+
+This release introduces new functionalities of CDA-telematics that includes the capability to:  
+
+1. Allow the ROS 2 NATS bridge and Kafka NATS bridge to stream data from the XiL environment. 
+2. Integrate the telematics tool as a plugin in V2X Hub to stream data from infrastructure units connected to V2X Hub. 
+3. Enable visualization of historical ROS 2 data recorded in rosbags  of the .mcap format by allowing upload, processing and writing of data from ROS 2 rosbags to the database from where it can be used to generate plots on the dashboard. 
+Note: The carma-system-4.5.0 release, which is the latest release for CARMA Platform - the main CDA system used to test and evaluate the telematics tool, currently does not support recording of rosbags in the .mcap format. Evaluation for historical data visualization was conducted using rosbags recorded on an unreleased version of CARMA Platform to enable recording of rosbags in the .mcap format. 
```

Comment:
This note can be removed. The feature exists on carma-system-4.5.0 https://github.com/usdot-fhwa-stol/carma-platform/commits/carma-system-4.5.0/carma/launch/ros2_rosbag.launch.py

Reply:
removed